src/AppBundle/Charts/Charts.php

```diff
@@ -0,0 +1,132 @@
+<?php
```

Comment:
http://symfony.com/doc/2.8/service_container.html
For exemple, in the "Referencing (Injecting) Services" section, we can see in the code.
1 // src/AppBundle/Newsletter/NewsletterManager.php
2 namespace AppBundle\Newsletter;